node_modules/node-red-contrib-middleware/nodes/5-httppost.html

```diff
@@ -0,0 +1,114 @@
+  </div>
+  <div class="form-row">
+    <label data-i18n="label.endpoint"></label>
+    <input type="text" id="node-input-endpoint" style="width: 70%"/>
+  </div>
+
+  <div class="form-row">
+      <label data-i18n="label.content"></label>
+      <input type="text" id="node-input-content" style="width: 70%"/>
+    </div>
+
+  <div class="form-row node-input-header-container-row">
+    <label for="node-input-name"><i class="fa fa-tag"></i> <span data-i18n="label.headers"></span></label>
+    <ol id="node-input-header-container"></ol>
+  </div>
+</script>
+
+<script type="text/x-red" data-help-name="http post">
+  <p>Sends responses back to http requests received from an HTTP Input node.</p>
+</script>
+
+<script type="text/javascript">
+
+  RED.nodes.registerType('http post', {
+    category: 'output',
+    color: "rgb(231, 231, 0)",
+    defaults: {
+      name: { value: "" },
+      endpoint: { value: "", required: true },
+      content: { value: "", required: true },
+      headers: { value: []}
+    },
+    inputs: 1,
+    outputs: 0,
+    align: "right",
+    icon: "white-globe.png",
+    label: function () {
+      return this.name || "http";
+    },
+    labelStyle: function () {
+      return this.name ? "node_label_italic" : "";
+    },
+    oneditprepare: function () {
+      $("#node-input-header-container").css('min-height', '250px').css('min-width', '450px').editableList({
+        addItem: function (container, i, data) {
+          if (!data.hasOwnProperty('header')) {
+            data.header = "";
+          }
+          var row = $('<div/>').appendTo(container);
+          var valueField = $('<input/>', { class: "node-input-header-value", type: "text", style: "margin-left: 5px;" }).typedInput({ default: 'str', types: ['str'] }).appendTo(row);
+          valueField.typedInput('value', data.header);
+
+          var finalspan = $('<span/>', { style: "float: right;margin-top: 6px;" }).appendTo(row);
+        },
+
+        removeItem: function (opt) {
+          if (opt.hasOwnProperty('i')) {
+            var removedList = $("#node-input-header-container").data('removedList') || [];
+            removedList.push(opt.i);
+            $("#node-input-header-container").data('removedList', removedList);
+          }
+          var headers = $("#node-input-header-container").editableList('items');
+          headers.each(function (i) { $(this).find(".node-input-header-index").html(i + 1); });
+        },
+        sortable: false,
+        removable: true
+      });
+
+
+      $("#node-input-content").typedInput({default:'msg',types:['msg']});
+
+      if (this.firstTime == undefined) {
+        $("#node-input-header-container").editableList('addItem', { header: 'Content-Type:application/json' });
+        $("#node-input-header-container").editableList('addItem', { header: 'Fiware-Service:service' });
+        $("#node-input-header-container").editableList('addItem', { header: 'Fiware-ServicePath:/' });
```

Comment:
Are those fiware headers really necessary?
Or better yet: why not use an already existent node definition from node-red?
(with a fully-configurable list of headers)